packages/gasket-plugin-command/test/command.test.js

```diff
@@ -1,16 +1,13 @@
-/* eslint-disable no-process-env */
-const assume = require('assume');
+/* eslint-disable no-process-env,jest/no-done-callback,jest/expect-expect,jest/no-conditional-expect */
```

Comment:
Can we clean up the test code to address some of these rules so we don't have to disable?

Reply:
I was able to remove one. The try catch assertion is unavoidable IMO. I can leave the warnings if you'd prefer.
```
   56:23  warning  Unexpected use of process.env          no-process-env
   57:7   warning  Unexpected use of process.env          no-process-env
   75:9   warning  Unexpected use of process.env          no-process-env
   81:5   warning  Unexpected use of process.env          no-process-env
  118:5   warning  Test has no assertions                 jest/expect-expect
  202:9   error    Avoid calling `expect` conditionally`  jest/no-conditional-expect
```

Reply:
`no-process-env` and `jest/expect-expect` are ok to disable here.

We should avoid the conditional assets/expects however. I believe you could adjust the one in this file like this:
```
await expect(async () => await cmd.run()).rejects.toThrow('The `gasketRun` method must be implemented');
```